README.md

```diff
@@ -176,17 +176,34 @@ for a specific value, e.g.
 
 - `regex`: matches the `actual` value when provided an `expected-regex` using `(re-find expected-regex actual)`
 
-- `absent`: for use in the context of maps. Matches when the actual map is missing the key pointing to the `absent` matcher. For example `(is (match? {:a absent :b 1} {:b 1}))` matches but `(is (match? {:a absent :b 1} {:a 0 :b 1}))` won't.
-
 - `match-with`: overrides default matchers for `expected` (scalar or arbitrarily deep stucture) (see Overriding default matchers, below)
 
 - `within-delta`: matches numeric values that are within `expected` +/- `delta` (inclusive)
 
+#### negative matchers
+
+Negative matchers, that is, those asserting the absence of something, are generally discouraged due to the negative effects they can have on code readability.
+
+- `mismatch`: Negation matcher that takes in an `expected` matcher and passes when it doesn't match the `actual`. For example, to assert the absence of an entry in a list `(is (match? (mismatch (embeds [odd?])) actual))`. Considering the mental burden of reasoning about negation, please use sparingly.
+- `absent`: for use in the context of maps. Matches when the actual map is missing the key pointing to the `absent` matcher. For example `(is (match? {:a absent :b 1} {:b 1}))` matches but `(is (match? {:a absent :b 1} {:a 0 :b 1}))` won't.
+
+##### readability concerns with negation matchers
+
+```clojure
+(deftest avoid-negative-matchers
+  (testing "normal assertion that `:a` is present"
+    (match? {:a any?}
+            actual))
+  (testing "double negation version"
+    (match? (matcher-combinators.matchers/mismatch {:a matcher-combinators.matchers/absent})
+            actual)))
+```
```

Comment:
how does this look/read?

Reply:
I can't make a suggestion because it wraps triple "`" in triple "`" :/

How about "are generally discouraged due to the adverse effect they can have on code readability."?

Also, I think we should recommend `:a-key nil` instead of `:a-key absent` except where the actual presence of the key really matters (which is almost never IMO).